libsql/src/replication/client.rs

```diff
@@ -58,7 +59,15 @@ impl Client {
             .try_into()
             .context("Invalid auth token must be ascii")?;
 
-        let ns = split_namespace(origin.host().unwrap()).unwrap_or_else(|_| "default".to_string());
+        let ns = if let Some(ns_from_arg) = maybe_namespace
+        {
```

Comment:
Nit: this `{` should probably be at the end of the previous line

Reply:
could that be configured so cargo fmt fixes it automatically?

Reply:
I would hope cargo fmt to fix it by default.